tests/integration/tests/menu.js

```diff
@@ -61,14 +61,15 @@ test('menu open favorite', async () => {
   expect(favorites).not.toBeNull();
 });
 
-test('one panel open at a time', async () => {
+test('close service panel when clicking direction', async () => {
   expect.assertions(2);
+  await page.goto(APP_URL);
   await page.click('.service_panel__item__direction');
   const servicePanelClose = await page.waitForSelector('.service_panel', { hidden: true });
   expect(servicePanelClose).not.toBeFalsy();
```

Comment:
Si tu regardes la doc de Puppeteer pour `waitForSelector` https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagewaitforselectorselector-options, tu verras que le résultat attendu quand tu utilises `hidden: true` est `null`. Du coup il faut remplacer `not.toBeFalsy` par `toBeNull`.